src/test/util/json.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <test/util/json.h>
+
+#include <boost/test/unit_test.hpp>
+#include <string>
+
+#include <univalue.h>
+
+UniValue read_json(const std::string& jsondata)
+{
+    UniValue v;
+    if (!v.read(jsondata) || !v.isArray()) {
+        BOOST_ERROR("Parse error.");
```

Comment:
Might be better to use `Assert` in any case to avoid the dead, confusing and dangerous return? (Dangerous because it returns an empty array, potentially running no tests at all)